Add how_to/interactivity (#4762)
holoviz/panel: doc/how_to/interactivity/index.md

# Make your component interactive

> This section is divided into three parts that will help you add interactivity to your applications. The first part covers how to make your functions interactive, while the second part explains how to bind parameters and bound functions to components. The third part focuses on using `hvplot.interactive` to make your plots interactive. By the end of this section, you will have the tools to enhance your applications with interactivity.

Reviewer: ```suggestion
This section will help you add interactivity to your applications and workflow.
```

::::{grid} 1 2 2 3
:gutter: 1 1 1 2

> :::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1 sd-animate-grow50` Make your functions interactive

Reviewer: make octicons unique for this page

:link: bind_function
:link-type: doc

How to bind functions to widgets to make them interactive
:::

> :::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1 sd-animate-grow50` Binding on components

Reviewer: ```suggestion
:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1 sd-animate-grow50` Bind component parameters to widgets
```

:link: bind_component
:link-type: doc

How to bind parameters and bound functions on components
:::

> :::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1 sd-animate-grow50` Make your plots interactive

Reviewer: ```suggestion
:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1 sd-animate-grow50` Make interative data workflows
```

:link: hvplot_interactive
:link-type: doc

> How to use `hvplot.interactive` with widgets to make your plots interactive

Reviewer: ```suggestion
How to use `hvplot.interactive` with widgets to make your data workflows interactive
```
